File: tests/tests/files/urlmap/root.js
const { chrome1 } = ChromeUtils.importESModule("chrome://global/content/test/chrome1.mjs");
const { resource1 } = ChromeUtils.importESModule("resource://test/resource1.mjs");

const lazy = {};
ChromeUtils.defineESModuleGetters(lazy, {
  chrome2: "chrome://global/content/test/chrome2.mjs",
  resource2: "resource://test/resource2.mjs",
});

window.open("chrome://global/content/test/chrome1.html");
window.open("resource://test/resource1.html");

File: tests/tests/files/urlmap/root.cpp
/**
 * chrome://global/content/test/chrome1.mjs
 * https://bugzilla.mozilla.org/
 * resource://test/resource1.mjs
 * moz-src:///urlmap/mozsrc1.mjs
 */

// chrome://global/content/test/chrome1.css
// https://bugzilla.mozilla.org/
// resource://test/resource1.png
// moz-src:///urlmap/mozsrc1.mjs

void f() {
  const char* s = "chrome://global/content/test/chrome1.mjs";
  const char* t = "https://bugzilla.mozilla.org/";
  const char* u = "resource://test/resource1.png";
  const char* v = "moz-src:///urlmap/mozsrc1.mjs";
}

